src/js/components/modals/ModalHeading.js

```diff
@@ -20,7 +20,7 @@ ModalHeading.defaultProps = {
 };
 
 ModalHeading.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
Why isn't this required anymore?

Reply:
It was never really; I just got rid of the proptype warning